pkg/server/helpers.go

```diff
@@ -95,10 +95,16 @@ const (
 )
 
 const (
-	// sandboxMetadataLabel is label name that identify metadata of sandbox in CreateContainerRequest
-	sandboxMetadataLabel = "io.cri-containerd.sandbox.metadata"
-	// sandboxMetadataLabel is label name that identify metadata of container in CreateContainerRequest
-	containerMetadataLabel = "io.cri-containerd.container.metadata"
+	// containerKindLabel is a label key indicating container is sandbox container or application container
+	containerKindLabel = "io.cri-containerd.kind"
```

Comment:
minor nit: Can we add a constant prefix = "io.cri-containerd"
And then use this for all the constants ?
for eg: containerKindLabel= prefix + "kind"

Reply:
done